lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -444,10 +450,10 @@ __END__
 
 =head1 SYNOPSIS
 
-   ledgersmb-admin user list <db-uri>
-   ledgersmb-admin user create <db-uri> [options] 
-   ledgersmb-admin user delete <db-uri>
-   ledgersmb-admin user change <db-uri>
+   ledgersmb-admin user list <db-uri> <user>
```

Comment:
Where does the `<user>` get parsed from the arguments? Is it stored in a variable somewhere?

Reply:
`user list` doesn't use a user tag, it lists all users. Fixed